idseq_dag/__main__.py

```diff
@@ -28,7 +28,7 @@ def main():
                             dag_json=args.dag_json,
                             versioned_output=args.versioned_output)
         log.write("everything is awesome. idseq dag is valid~")
-    except:
+    except Exception:
```

Comment:
This change might be introducing a bug, because, if I understand correctly, only user-defined exceptions are subclasses of the Exception class;   runtime exceptions may not be.   The intent of this line is to catch every exception, and then re-raise it.   It's valid Python, and it's a linter bug to flag it.  My recommendation is to disable the lint warning.

Reply:
All exceptions except SystemExit are subclasses of Exception.